app/views/dashboard/index.html.erb

```diff
@@ -0,0 +1,65 @@
+<div class="container-fluid">
+  <div class="jumbotron">
+    <h1>Welcome, Job Hunter!</h1>
+  </div>
+  <hr>
+  <div class="row">
+    <div class="col-md-4">
+      <h3>Count of jobs by level of interest</h3>
+      <table class="table table-condensed table-striped">
+        <thead>
+          <tr>
+            <th><%= "Level of Interest" %></th>
+            <th><%= "Number of Jobs" %></th>
+          </tr>
+        </thead>
+        <tbody>
+          <% Job.count_by_level_of_interest.each_pair do |interest, count| %>
```

Comment:
i suggest you move this method call out of the view. it's the controller's job to grab your data and pass it to your views